packages/admin/src/pages/dashboard/ManageGrantSchemes.test.tsx

```diff
@@ -0,0 +1,135 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import CustomLink from '../../components/custom-link/CustomLink';
+import Scheme from '../../types/Scheme';
+import ManageGrantSchemes, {
+  generateSchemeTableRows,
+} from './ManageGrantSchemes';
+
+jest.mock('next/config', () => () => {
+  return {
+    serverRuntimeConfig: {
+      backendHost: 'http://localhost:8080',
+    },
+    publicRuntimeConfig: {
+      SUB_PATH: '/apply',
+      APPLICANT_DOMAIN: 'http://localhost:8080',
+    },
+  };
+});
```

Comment:
this mock is all over the repo but I think it's just been copy-pasted around, we already mock `next/server` in `setupJestMock.js`

Reply:
Nice one, cheers for the heads up

Reply:
sorted